dvc/repo/add.py

```diff
@@ -38,6 +40,26 @@ def add(
         raise RecursiveAddingWhileUsingFilename()
 
     targets = ensure_list(targets)
+
+    invalid_opt = None
+    if to_remote:
+        message = "{option} can't be used with --to-remote"
+        if len(targets) != 1:
+            invalid_opt = "multiple targets"
```

Comment:
Why can't you add multiple targets (or `--recursive`) straight-to-remote? (only the external ones or checking that all are external). If it's for simplicity of implementation for now, then I understand.

Reply:
> Why can't you add multiple targets (or --recursive) straight-to-remote? 

It was just for mirroring the behavior for `import-url` (which this actually initially implemented in), though it should be trivial to support adding multiple targets if we make `-o`/`--out` take multiple arguments and then figure out a way to match the outputs (like what happens if the user supplies less output destinations than the total number of targets, seems a bit ambigous since it is an optional value).

Reply:
I agree with @isidentical , let's not complicate (especially ambiguity around `-o` here) it for now. Let's wait for someone to ask about this.

Reply:
OK. I wasn't proposing to enable this, I was asking.